packages/cfd/src/Containers/cfd-compare-accounts/cfd-compare-accounts.tsx

```diff
@@ -64,19 +64,20 @@ const CompareCFDs = observer(() => {
             ? all_cfd_available_accounts.length + 1
             : all_cfd_available_accounts.length;
 
-    const CompareAccountsHeader = (
-        <Localize
-            i18n_default_text={
-                selected_region === REGION.EU
-                    ? 'Deriv MT5 CFDs {{real_title}} account'
-                    : 'Compare CFDs {{demo_title}} accounts'
-            }
-            values={{
-                demo_title: is_demo ? localize('demo') : '',
-                real_title: is_demo ? localize('Demo') : localize('real'),
-            }}
-        />
-    );
+    const getCompareAccountsHeader = () =>
+        selected_region === REGION.EU ? (
+            <Localize
+                i18n_default_text='Deriv MT5 CFDs {{title}} account'
+                values={{
+                    title: is_demo ? localize('Demo') : localize('real'),
```

Comment:
For title, the text case for Demo and real seem to be different. Is this expected?

Reply:
yes we have a difference in title in real and demo account according to design.

<img width="245" alt="Screenshot 2024-10-03 at 8 43 08 AM" src="https://github.com/user-attachments/assets/3c0bff95-3e02-426e-ab8a-5d926198a547">
<img width="232" alt="Screenshot 2024-10-03 at 8 44 10 AM" src="https://github.com/user-attachments/assets/0af7d233-1984-4e13-ad15-2fc65676142c">

Reply:
@Dhruv-deriv it should 'demo' not 'Demo'. i have updated it in code